packages/plugins/experimental/plugin-map/package.json

```diff
@@ -52,6 +52,7 @@
     "@dxos/plugin-graph": "workspace:*",
     "@dxos/plugin-space": "workspace:*",
     "@dxos/plugin-stack": "workspace:*",
+    "@dxos/plugin-table": "workspace:*",
```

Comment:
When I merged in `main` it broke the `TableType` import, added this as a dep to get it back.

Reply:
cc @richburdon